src/xdocs/config_coding.xml

```diff
@@ -4294,7 +4294,21 @@ for(int i=0; i&lt;10; i++) {
         <source>
 &lt;module name=&quot;NestedIfDepth&quot;/&gt;
         </source>
-
+        <p>Valid Code Example:</p>
+        <source>
+if (x &gt; 2) {
+    if (x &gt; 3) {}
+    else {} // OK
+}
+        </source>
+        <p>Invalid Code Example:</p>
+        <source>
+if (x &gt; 2) {
+    if (x &gt; 3) {}
+    if (x &gt; 4) {}
+    else {} // violation, depth exceeded '1'
```

Comment:
nesting is still one here.
because inside an if block there are two if blocks, but they are not nested.
see here: https://www.programiz.com/java-programming/if-else-statement

Reply:
done.

Reply:
resolved